firstvoices/backend/models/app.py

```diff
@@ -56,3 +56,24 @@ class Meta:
 
     def __str__(self):
         return f"{self.user} ({self.get_role_display()})"
+
+
+class AppImportStatus(BaseModel):
+    """Represents the status of manual data imports, such as nuxeo imports."""
+
+    label = models.CharField(max_length=150)
+    successful = models.BooleanField(default=False)
+    warnings = models.BooleanField(default=False)
+
```

Comment:
Optional: It might be useful to add one more field which could hold some warning or failure information. This could probably be done down the road when we add to the import script though.